db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170321170516) do
+ActiveRecord::Schema.define(version: 20170331194256) do
```

Comment:
If the DB changes were reverted, should this schema version change be reverted as well?